README.md

```diff
@@ -57,7 +57,13 @@ tensorboard --logdir ~/.cache/torchok/logs/cifar10
 ### Export to ONNX
 TODO
 ### Test ONNX model
-TODO
+For test example we can use the default configuration `examples/configs/onnx_infer.yml`.
```

Comment:
Is it start? Because all configs extension is `yAml` anyway change it to yaml please

Reply:
I think it would be better if we say
`For the ONNX model test, we can use the` because we also can test ckpt also.